docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
+
+When you have the repository cloned locally, set up your local environment:
+
+1. [Install Node.js](https://nodejs.org).
+
+1. At a command line prompt, navigate to the root of the repository:
+
+   `cd path/to/web-features`
+
+1. Install the project dependencies:
+
+   `npm install`
+
+1. Create a new Git branch to track your work:
+
+   `git checkout -b <your branch name>`
+
+### Create a new feature from scratch
+
+To create a new feature from scratch:
+
+1. Go through the steps in [Fork the repository and set up your local environment](#fork-the-repository-and-set-up-your-local-environment).
+
+1. Decide what the unique ID for your new feature should be. To help you, check the [Identifiers](./guidelines.md#identifiers) section of the feature guidelines.
+
+1. Create a file in the `features` directory with the name of your feature ID, and the `.yml` extension. For example, if your feature ID is `my-feature`, create a file named `features/my-feature.yml`.
+
+1. Open the file in your favorite text editor, and add the following minimum content:
+
+   ```yaml
+   name: "My feature"
+   description: "A short description of the feature."
+   spec: https://urlofthespec.com
+   ```
+
+   For guidance, see the [Names](./guidelines.md#names) and [Descriptions](./guidelines.md#descriptions) sections of the feature guidelines.
+
+1. Optionally add a `group` field to the feature.
+
+   The `group` field is used to categorize features into groups. For example, the Async Clipboard API feature is in the `Clipboard` group. Groups are maintained in the [`groups`](https://github.com/web-platform-dx/web-features/tree/main/groups) directory, and each group is a YAML file.
+
+   * If one of the existing groups fits your feature, add the `group` field to your feature, and set it to the name of the group.
+   * If none of the existing groups fit your feature, but you believe a new group should be greated, then create a new group file in the `groups` directory, and add the feature to that group.
+
+1. Set the baseline status and browser compatibility data of the feature. This is the most important and difficult step of authoring a feature, which is documented in a separate section. See [Set the status of a feature](#set-the-status-of-a-feature).
+
+### Create a new feature from a draft feature
+
+To start from an existing draft feature:
+
+1. Go through the steps in [Fork the repository and set up your local environment](#fork-the-repository-and-set-up-your-local-environment).
+
+1. Review the existing draft features by looking at the YAML files in the [`features/draft`](https://github.com/web-platform-dx/web-features/tree/main/features/draft) directory and sub-directories.
+
+1. Find a draft that you want to work on.
+
+1. Move the file to the `features` directory.
+
+1. Remove the `draft_date` field from the file.
+
+1. Review the feature ID, name, description, and spec fields.
```

Comment:
Consider splitting `description` here. Most draft features have `TODO` as a description. That requires more than a review!